### PR TITLE
TaskRunner draft

### PR DESCRIPTION
**Pull request by amgassert**

### CODE REVIEW

File: src/task-runner.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const config = require('../config.json');
+const Discord = require('discord.js');
+const client = new Discord.Client();
+
+let TaskRunner = {
+    addUser: function(message) {
+        let role = message.guild.roles.find(role => role.name === message.channel.name);
+        message.member.addRole(role);
```
Comment (halawami) [inline src/task-runner.js]: Should we make this more defensive in case they put a non existent role or member?

**Comment by CelticMajora** [comment]
Is this still a valid PR?

File: src/context-manager.js [hunk 2]
```diff
@@ -99,7 +99,99 @@ let ContextManager = {
 				userContext.user_context = user_context
 			}
 		}
+<<<<<<< HEAD
```
Comment (CelticMajora) [inline src/context-manager.js]: These functions can all be taken out and the merge conflict tokens removed

File: src/endpoints.js [hunk 3]
```diff
@@ -1,32 +1,55 @@
 const config = require("../config.json");
+<<<<<<< HEAD
```
Comment (CelticMajora) [inline src/endpoints.js]: We will need all of these, just pull out the merge conflict tokens

File: src/endpoints.js [hunk 4]
```diff
@@ -1,32 +1,55 @@
...
 const contextManager = require("./context-manager.js");
+>>>>>>> gh-pages
 
 function messageHandler(message){
 	if(message.author.bot) return;
 	if(message.content.indexOf(config.prefix) !== 0) return;
 
 	const args = message.content.slice(config.prefix.length).trim().split(/ +/g);
 	const command = args.shift().toLowerCase();
+<<<<<<< HEAD
```
Comment (CelticMajora) [inline src/endpoints.js]: Similar thing here, just take out the tokens.

File: src/context-manager.js (hunk 2 above)
Comment (amgassert) [reply src/context-manager.js]: Odd. This stuff isn't in my local copy, and it looks like I've already pushed everything.

Comment (CelticMajora) [reply src/context-manager.js]: Hmm. Try committing maybe. Resolving conflict can be a bit wacky with git.

Comment (amgassert) [reply src/context-manager.js]: Already done. Will spend some more time investigating tonight.

File: src/task-runner.js (hunk 1 above)
Comment (amgassert) [reply src/task-runner.js]: Now checking in endpoints

Comment (amgassert) [reply src/task-runner.js]: Now checking in endpoints

**Pull request merged**